docs/docs/tutorial.md

```diff
@@ -549,7 +549,7 @@ var CommentForm = React.createClass({
     var author = this.state.author.trim();
     var text = this.state.text.trim();
     if (!text || !author) {
-      return;
+      return false;
```

Comment:
[Here's a jsfiddle](https://jsfiddle.net/69z2wepo/46290/) with the current code, I'm not seeing any difference when the `return` value is changed. What version of React are you using?